### DISCUSSION TITLE
Polestar expidition bug?

### DISCUSSION THREAD

**Discussion by gitdp7885** [leodium/leodium-ReLive-NoMansSky-Expeditions-Offline#27] [Q&A] [answered]
The C class warp core upgrade module for the freighter warp drive seens to want cadmium rather than chromatic metal.  Since you need the C class upgradre to get to a red star this seems like a mistake somewhere.  (Playing after Worlds part 1 and the fishing expidition.

**Comment by cwmonkey** [comment]
@gitdp7885 Nope, that looks correct: https://nomanssky.fandom.com/wiki/Amplified_Warp_Shielding

Also, this isn't something our .json files would affect, especially after you've finished the expedition.

**Reply by gitdp7885** [reply, marked as answer]
OK, my bad.
